.config/.prettierignore

```diff
@@ -1,13 +1,14 @@
 **/js/app/public/**
 **/pnpm-workspace.yaml
 **/js/app/dist/**
+**/client/js/dist/**
 **/pnpm-lock.yaml
 **/js/plot/src/Plot.svelte
 **/.svelte-kit/**
 **/demo/**
 **/gradio/**
 **/website/**
-**/venv/**
+**/.venv/**
```

Comment:
The `venv` directory [configured in `.gitignore`](https://github.com/gradio-app/gradio/blob/3227a9b703fb520bf4f8a8ee08f9027c0a5c1070/.gitignore#L53) is `.venv`.